src/api/.rubocop.yml

```diff
@@ -92,6 +92,9 @@ Lint/AssignmentInCondition:
 Lint/EndAlignment:
   AlignWith: variable
 
+Lint/Eval:
```

Comment:
But it is not like that for all cops and to be consistent I would add it for all of them. It is not a lot work and it may be helpful. :slightly_smiling_face: